.github/workflows/build.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{fromJson(needs.pre-filter.outputs.matrix)}}
-    timeout-minutes: 15
+    timeout-minutes: 20
```

Comment:
This is crazy...

Reply:
I will merge the squashed migration branch very soon. But we need to organize these tests differently and move several out of notebooks.